src/routes/applet/change_of_basis/projection/+page.svelte

```diff
@@ -24,15 +24,15 @@
 <Canvas3D>
   <!-- Vector b1 -->
   <Vector3D direction={b1} length={b1.length()} color={PrimeColor.ultramarine} />
-  <Latex3D latex={'\\bold{b1}'} position={b1} />
+  <Latex3D latex={'\\bold{b_1}'} position={b1} color={PrimeColor.ultramarine}/>
```

Comment:
```suggestion
  <Latex3D latex={'\\bold{b}_1'} position={b1} color={PrimeColor.ultramarine}/>
```

and fix for all other instances where \bold{x_1} to \bold{x}_1